src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/HeartbeatTask.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks;
+
+import net.javacrumbs.shedlock.core.SchedulerLock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.msg.HeartbeatMsg;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.servicebus.ServiceBusHelper;
+
+/**
+ * Sends heartbeat messages to orchestrator
+ */
+@Component
+public class HeartbeatTask {
+
+    private static final Logger log = LoggerFactory.getLogger(HeartbeatTask.class);
+
+    private final ServiceBusHelper serviceBusHelper;
+
+    public HeartbeatTask(@Qualifier("envelopes-helper") ServiceBusHelper serviceBusHelper) {
+        this.serviceBusHelper = serviceBusHelper;
+    }
+
+    @Scheduled(cron = "0 0/10 * * * *")
```

Comment:
unless we want to have different setting of different envs I think this can stay as it is.